src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -60,6 +60,19 @@ public DatastoreScheduledInterviewDao() {
     return Optional.of(entityToScheduledInterview(scheduledInterviewEntity));
   }
 
+  /**
+   * Returns a list of all scheduled ScheduledInterview objects whose startTime is between minTime
```

Comment:
No need to say scheduled ScheduledInterview. minTime and maxTime are not in milliseconds, and just say 'objects between...', don't bring startTime into it.